core/arch/aarch64/encode.c

```diff
@@ -79,6 +79,8 @@ const char * const reg_names[] = {
     "z8", "z9", "z10", "z11", "z12", "z13", "z14", "z15",
     "z16", "z17", "z18", "z19", "z20", "z21", "z22", "z23",
     "z24", "z25", "z26", "z27", "z28", "z29", "z30", "z31",
+    "p0", "p1", "p2", "p3", "p4", "p5", "p6", "p7",
+    "p8", "p9", "p10", "p11", "p12", "p13", "p14", "p15",
```

Comment:
This order no longer matches the enum

Reply:
Done thanks. I've moved everything to the end.